packages/controlled-form/src/lib/ControlledForm.tsx

```diff
@@ -0,0 +1,29 @@
+import { FormHTMLAttributes } from 'react';
+import { useForm, SubmitHandler, FormProvider, Resolver } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as yup from 'yup';
```

Comment:
do we want to lock everyone into yup? i thought we weren't touching the resolvers so everyone could choose which one they want and pass the resolved schema to the controlled form.